server/container/guice/memory-guice/src/main/java/org/apache/james/MemoryJamesServerMain.java

```diff
@@ -74,6 +75,7 @@
         new DeletedMessageVaultRoutesModule(),
         new DLPRoutesModule(),
         new MailboxesExportRoutesModule(),
+        new CreateMissingParentsRoutesModule(),
```

Comment:
I prefer to `install` this module within MailboxRoutesModule. It would be so easy to forget it otherwise....